main.tf

```diff
@@ -1,3 +1,9 @@
+module "crn_parser" {
+  source  = "terraform-ibm-modules/common-utilities/ibm//modules/crn-parser"
+  version = "1.1.0"
+  crn     = var.kms_key_crn
```

Comment:
`kms_key_crn` is optional. Meaning it could have a value of `null`. Meaning this module call would fail. You need to add a count around it to ensure its only called if `kms_key_crn` is not null